types/react-native/test/index.tsx

```diff
@@ -891,6 +891,7 @@ class ScrollerListComponentTest extends React.Component<{}, { dataSource: ListVi
                             fadingEdgeLength={200}
                             StickyHeaderComponent={this._stickyHeaderComponent}
                             stickyHeaderHiddenOnScroll={true}
+                            automaticallyAdjustKeyboardInsets
```

Comment:
here in the tests can you also add a separate

```
const ScrollViewInsetsTest = () => (
  <>
    <ScrollView automaticallyAdjustKeyboardInsets />
    <ScrollView automaticallyAdjustKeyboardInsets={false} />
  </>
);
```
that would match the one that we had previously in place